src/client/components/features/FormationInitiale/Rechercher/RechercherFormationInitiale.test.tsx

```diff
@@ -45,19 +45,71 @@ describe('RechercherFormationInitiale', () => {
 			expect(aFormationService.rechercherFormationInitiale).toHaveBeenCalledWith({ motCle: 'boulanger' });
 		});
 
-		it('lorsqu‘il y a plusieurs résultats je vois le nombre de résultats affiché', async () => {
-			const aFormationService = aFormationInitialeService();
-
-			const resultRechercheFormation = createSuccess([aResultatListFormationInitiale({ libelle: 'boulanger' }),
-				aResultatListFormationInitiale({ libelle: 'patissier' })]);
-			jest.spyOn(aFormationService, 'rechercherFormationInitiale').mockResolvedValue(resultRechercheFormation);
-
-			render(<DependenciesProvider formationInitialeService={aFormationService}>
-				<RechercherFormationInitiale/>
-			</DependenciesProvider>,
-			);
-
-			expect(await screen.findByText(/[0-9]+ formations pour boulanger/)).toBeVisible();
+		describe('lorsqu‘il y a au moins un résultat', () => {
+			describe('le nombre de résultat', () => {
+				it('lorsqu‘il y a un résultat je vois le nombre de résultats affiché au singulier', async () => {
+					const aFormationService = aFormationInitialeService();
+					const resultRechercheFormation = createSuccess([aResultatFormationInitiale({ libelle: 'boulanger' })]);
+					jest.spyOn(aFormationService, 'rechercherFormationInitiale').mockResolvedValue(resultRechercheFormation);
+
+					render(<DependenciesProvider formationInitialeService={aFormationService}>
+						<RechercherFormationInitiale/>
+					</DependenciesProvider>,
+					);
+
+					expect(await screen.findByText(/1 formation/)).toBeVisible();
+				});
+				it('lorsqu‘il y a plusieurs résultats je vois le nombre de résultats affiché', async () => {
+					const aFormationService = aFormationInitialeService();
+
+					const resultRechercheFormation = createSuccess([aResultatFormationInitiale({ libelle: 'boulanger' }),
+						aResultatFormationInitiale({ libelle: 'patissier' })]);
+					jest.spyOn(aFormationService, 'rechercherFormationInitiale').mockResolvedValue(resultRechercheFormation);
+
+					render(<DependenciesProvider formationInitialeService={aFormationService}>
+						<RechercherFormationInitiale/>
+					</DependenciesProvider>,
+					);
+
+					expect(await screen.findByText(/[0-9]+ formations pour boulanger/)).toBeVisible();
```

Comment:
nitpick: comme on a pas de test qui vérifie que le nombre de formations correspond bien au nombre de résultats dans la liste, on aurait pu mettre : 
```suggestion
					expect(await screen.findByText(/2 formations pour boulanger/)).toBeVisible();
```